src/utils/components/SSHSecretSection/components/SSHOptionUseExisting/SSHOptionUseExisting.tsx

```diff
@@ -0,0 +1,168 @@
+  setSSHDetails: Dispatch<SetStateAction<SSHSecretDetails>>;
+  sshDetails: SSHSecretDetails;
+};
+
+const SSHOptionUseExisting: FC<SSHOptionUseExistingProps> = ({
+  namespace,
+  projectsWithSecrets,
+  setSSHDetails,
+  sshDetails,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [selectedProject, setSelectedProject] = useState<string>();
+  const projects = useMemo(() => Object.keys(projectsWithSecrets), [projectsWithSecrets]);
+  const showNewSecretNameField = namespace
+    ? selectedProject !== namespace
+    : selectedProject !== sshDetails?.sshSecretNamespace;
+
+  useEffect(
+    () =>
+      !selectedProject &&
+      setSelectedProject(namespace || sshDetails?.sshSecretNamespace || projects?.[0]),
+    [namespace, projects, selectedProject, sshDetails?.sshSecretNamespace],
+  );
+
+  const onFilterProject = (_: any, value: string) => {
+    const filteredProjects = projects?.filter((project) => project.includes(value));
+
+    return filteredProjects?.map((project) => (
+      <SelectOption key={project} value={project}>
+        <ResourceLink
+          groupVersionKind={modelToGroupVersionKind(ProjectModel)}
+          linkTo={false}
+          name={project}
+        />
+      </SelectOption>
+    ));
+  };
+
+  const onSelectProject = useCallback(
+    (_: any, newValue: string) => {
+      setSelectedProject(newValue);
+      const addNew = namespace ? newValue !== namespace : newValue !== activeNamespace;
+      setSSHDetails((prev) => ({
+        ...prev,
+        secretOption: addNew ? SecretSelectionOption.addNew : SecretSelectionOption.useExisting,
+        sshPubKey: '',
+        sshSecretName: '',
+        sshSecretNamespace: newValue,
+      }));
+
+      setIsOpen(false);
+    },
+    [setSSHDetails, activeNamespace, namespace],
+  );
+
+  const onChangeSecretName = (newSecretName: string) => {
+    setSSHDetails((prev) => ({
+      ...prev,
+      sshSecretName: newSecretName,
+    }));
+    setIsOpen(false);
+  };
+
+  return !isEmpty(projects) ? (
+    <>
+      <Alert
+        title={t(
+          'Select a secret from a different project to copy the secret to your current project.',
+        )}
+        isInline
+        variant={AlertVariant.info}
+      />
+      <Grid className="ssh-secret-section__body">
+        <GridItem span={6}>
+          <FormGroup fieldId="project" label={t('Project')}>
+            <Select
+              className="ssh-secret-section__form-group-project"
+              hasInlineFilter
+              inlineFilterPlaceholderText={t('Search project')}
+              isOpen={isOpen}
+              maxHeight={400}
+              onFilter={onFilterProject}
+              onSelect={onSelectProject}
+              onToggle={setIsOpen}
+              selections={selectedProject}
+              variant={SelectVariant.single}
+            >
```

Comment:
use 
```
menuAppendTo="parent"
````
to fix selects in modals that get hided

Reply:
cool, just checked it does make a different, will push